models/topics-model.js

```diff
@@ -34,3 +34,20 @@ exports.fetchArticleById = async (id) => {
     });
   return rows[0];
 };
+
+exports.fetchAllArticles = async () => {
+  const { rows } = await db.query(
+    `
+      SELECT article_id, title, topic, author, created_at, votes, article_img_url FROM articles ORDER BY created_at DESC
+     `
+  );
+  const articlesArray = rows;
+  for (const article of articlesArray) {
+    const comments = await db.query(
+      `SELECT COUNT(*) FROM comments WHERE article_id = $1`,
+      [article.article_id]
+    );
```

Comment:
We want to do as few db.queries as possible. Can you think of a way to do both of these in one query string/